readme.md

```diff
@@ -617,13 +618,11 @@
 - [Agriculture](https://github.com/beaorn/awesome-agriculture#readme) - Open source technology for farming and gardening.
 - [Plotters](https://github.com/beardicus/awesome-plotters#readme) - Computer-controlled drawing machines and other visual art robots.
 
-
 ## Related
 
 - [Awesome Indexed](https://awesome-indexed.mathew-davies.co.uk) - Search the Awesome dataset.
 - [Awesome Search](https://awesomelists.top) - Quick search for Awesome lists.
 
-
```

Comment:
Don't do unrelated changes.

Reply:
must have happened as a result of rebasing/merging the diff since i initially opened the PR in january. in any case, it's fixed now.